CONTRIBUTING.md

```diff
@@ -13,6 +13,19 @@ To get started with the project, run `yarn install` in the root directory to ins
 yarn install
 ```
 
+### Project structure overview
+
+Repository contains 2 main projects:
+
+* SDK project, which consists of 3 workspaces: `react-native-sdk`, `react-native-sdk-react-navigation`, `react-native-sdk-react-native-navigation`
+* Sample app project (in the `example` folder)
+
+Sample app project exists as dedicated project, because React Native tooling (`Metro` bundler specifically) [doesn't support symbolic links](https://github.com/facebook/metro/issues/1) which is utilized by the [yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) to have a single places for hosting `node_modules`. On the other hand, having symbolic links for SDK modules works fine, because it doesn't use React Native build tooling for running tests and linting.
+
+Workspaces are managed with [Lerna](https://github.com/lerna/lerna).
+
+If you are using Visual Studio Code, please make sure to set workspace version of TypeScript instead of bundled with Visual Studio, check [this](https://stackoverflow.com/a/39676463/3222695) to see how to do it.
```

Comment:
```suggestion
If you are using Visual Studio Code, set your workspace version to TypeScript instead of the native JS bundled with Visual Studio, check [this](https://stackoverflow.com/a/39676463/3222695) to see how to do it.
```

Reply:
What is meant here is to use proper version of TypeScript instead of TypeScript version bundled with VS Code (not JS version).

Reply:
updated the suggestion to say Typescript